apparmor.d/abstractions/bus/org.freedesktop.PackageKit

```diff
@@ -2,6 +2,41 @@
 # Copyright (C) 2023 Alexandre Pujol <alexandre@pujol.io>
 # SPDX-License-Identifier: GPL-2.0-only
 
+  dbus receive bus=system path=/@{int}@{int}@{int}@{int}@{int}_edcedacc
+       interface=org.freedesktop.PackageKit.Transaction
+       member=GetUpdates
+       peer=(name=:*),
+
+  dbus receive bus=system path=/@{int}@{int}@{int}@{int}@{int}_edcedacc
+       interface=org.freedesktop.DBus.Properties
+       member=GetUpdates
+       peer=(name=:*),
+
+  dbus receive bus=system path=/org/freedesktop/PackageKit
+       interface=org.freedesktop.DBus.Introspect
+       member=Introspect
+       peer=(name=:*),
+
+  dbus send bus=system path=/@{int}@{int}@{int}@{int}@{int}_edcedacc
+       interface=org.freedesktop.PackageKit.Transaction
+       member={Destroy,Finished,Package}
+       peer=(name=:*),
+
+  dbus send bus=system path=/@{int}@{int}@{int}@{int}@{int}_edcedacc
+       interface=org.freedesktop.PackageKit.Properties
+       member=PropertiesChanged
+       peer=(name=:*),
+
+  dbus send bus=system path=/org/freedesktop/DBus
+       interface=org.freedesktop.DBus
+       member={AddMatch,GetNameOwner,Hello,RequestName}
+       peer=(name=:*),
```

Comment:
Remove:
1. Not the place for this
2. Already included in `bus-system`
3. Always specify the target program profile with the peer label

Reply:
Because parts of KDE aren't confined yet, and some parts are started via shell and run unconfined, some peer labels are `unconfined`. What do I do in such a situation?